internal/cmd/server/create_image.go

```diff
@@ -15,7 +15,7 @@ import (
 var CreateImageCmd = base.Cmd{
 	BaseCobraCommand: func(client hcapi2.Client) *cobra.Command {
 		cmd := &cobra.Command{
-			Use:   "create-image [FLAGS] SERVER",
+			Use:   "create-image [options] <server>",
```

Comment:
```suggestion
			Use:   "create-image [options] <server> --type <snapshot|backup>",
```

`--type` is required.